js/map.js

```diff
@@ -6,11 +6,43 @@ $(window).resize(function () {
 }).resize();
 
 $(function() {
+  // https://github.com/Aleksander98/bsgdprcookies
+  // settings for jquery.bs.gdpr.cookies.js
+  var settings = {
+      title: 'About our cookies',
+      message: 'Probation Community Resources uses cookies. Cookies enable the "Save search" functionality, which lets you save your favorite searches for easy retrieval: DataMade does not collect or use this data in any way. By using this site you consent to our cookie policy.',
```

Comment:
Let's reduce this—

>Probation Community Resources uses cookies to allow you to save your searches for easy retrieval. DataMade does not collect or use this data in any way. By using this site you consent to our cookie policy.

Can we also lower the font of the message text? It seems too large right now.